### ISSUE TITLE
`cbind_rtables` loses formats of individual tables that are combined

### ISSUE THREAD

**Issue by danielinteractive** [Roche/rtables#112]
```r
library(rtables)

tab1 <- rtable(
  header = "a",
  rrow("one", 1.1111111, format = "xx.x")
)
tab2 <- rtable(
  header = "b",
  rrow("one", 2.2222222, format = "xx.xxxx")
)

wrong_format <- cbind_rtables(tab1, tab2)
```

It seems like only the format of the left table is used, so that the formats of the right table are overwritten wrongly by the left table formats. We would want the formats to stay correct per column/sub table. (same session info here as in #111)

**Comment by gmbecker** [comment]
If you want the formats to stay on a per cell basis they should be on the
cell level. A row can only ever have one format associated with it, by
design.

That said there is currently a big where what should work, ie

```
tab1 <- rtable(
  header = "a",
  rrow("one", rcell(1.1111111, format = "xx.x"))
)
tab2 <- rtable(
  header = "b",
  rrow("one", rcell(2.2222222, format = "xx.xxxx"))
)

cbind_rtables(tab1, tab2)
```

also doesn't work because the cell formats are being put on the rows, but I
will get that fixed.

**Comment by danielinteractive** [comment]
Thanks @gmbecker . Will be great to have this fixed.

**Comment by gmbecker** [comment]
This is fixed in 24a972c but that unearthed a bug, which is now fixed in 13f32f1
 
Make sure you use version 0.3.2.17.9035, from the latter commit, to get both this and the unearthed bug fixed.

Thanks for the report.

**Comment by danielinteractive** [comment]
Thanks a lot @gmbecker !
